src/Themes/NexusMods.Themes.NexusFluentDark/Styles/Controls/Icons/IconsStyles.axaml

```diff
@@ -1,490 +1,347 @@
-        <Setter Property="Value">
-            <unifiedIcon:IconValue MdiValueSetter="mdi-drag-vertical-variant" />
-        </Setter>
+    <Style Selector="icons|UnifiedIcon.DragVertical">
+        <Setter Property="Value" Value="{x:Static icons:IconValues.DragHandleVertical}"/>
     </Style>
 
-    <Style Selector="unifiedIcon|UnifiedIcon.Search">
-        <Setter Property="Value">
-            <unifiedIcon:IconValue MdiValueSetter="mdi-magnify" />
-        </Setter>
+    <Style Selector="icons|UnifiedIcon.Search">
+        <Setter Property="Value" Value="{x:Static icons:IconValues.Search}"/>
     </Style>
 
-    <Style Selector="unifiedIcon|UnifiedIcon.Home">
-        <Setter Property="Value">
-            <unifiedIcon:IconValue MdiValueSetter="mdi-home" />
-        </Setter>
+    <Style Selector="icons|UnifiedIcon.Home">
+        <Setter Property="Value" Value="{x:Static icons:IconValues.Home}"/>
     </Style>
 
-    <Style Selector="unifiedIcon|UnifiedIcon.ToggleSwitch">
-        <Setter Property="Value">
-            <unifiedIcon:IconValue MdiValueSetter="mdi-toggle-switch-outline" />
-        </Setter>
+    <Style Selector="icons|UnifiedIcon.ToggleSwitch">
+        <Setter Property="Value" Value="{x:Static icons:IconValues.ToggleOn}"/>
     </Style>
 
-    <Style Selector="unifiedIcon|UnifiedIcon.ToggleSwitchOff">
-        <Setter Property="Value">
-            <unifiedIcon:IconValue MdiValueSetter="mdi-toggle-switch-off-outline" />
-        </Setter>
+    <Style Selector="icons|UnifiedIcon.ToggleSwitchOff">
+        <Setter Property="Value" Value="{x:Static icons:IconValues.ToggleOff}"/>
     </Style>
 
-    <Style Selector="unifiedIcon|UnifiedIcon.News">
-        <Setter Property="Value">
-            <unifiedIcon:IconValue MdiValueSetter="mdi-newspaper-variant-outline" />
-        </Setter>
+    <Style Selector="icons|UnifiedIcon.Game">
+        <Setter Property="Value" Value="{x:Static icons:IconValues.Game}"/>
     </Style>
 
-    <Style Selector="unifiedIcon|UnifiedIcon.Bookmark">
-        <Setter Property="Value">
-            <unifiedIcon:IconValue MdiValueSetter="mdi-bookmark-multiple-outline" />
-        </Setter>
+    <Style Selector="icons|UnifiedIcon.Undo">
+        <Setter Property="Value" Value="{x:Static icons:IconValues.Undo}"/>
     </Style>
 
-    <Style Selector="unifiedIcon|UnifiedIcon.Game">
-        <Setter Property="Value">
-            <unifiedIcon:IconValue MdiValueSetter="mdi-gamepad-square-outline" />
-        </Setter>
+    <Style Selector="icons|UnifiedIcon.Redo">
+        <Setter Property="Value" Value="{x:Static icons:IconValues.Redo}"/>
     </Style>
 
-    <Style Selector="unifiedIcon|UnifiedIcon.HourglassEmpty">
+    <Style Selector="icons|UnifiedIcon.AddCircle">
         <Setter Property="Value">
-            <unifiedIcon:IconValue MdiValueSetter="mdi-timer-sand-empty" />
+            <icons:IconValue SvgSetter="avares://NexusMods.App.UI/Assets/Icons/add_circle_24px.svg" />
```

Comment:
I think we may want to just add the GeometryData inline for this one and remove the icon from the project.
It's a bit of an outlier. Or we can leave it in as reference.

Reply:
I'm actually not sure if SVGs can be translated directly as `GeometryData`. It appears to work just fine but I'd like to investigate this at some point later.

Reply:
I think I exported that svg from figma, so it would be a matter of using the original icon Geometry data from figma instead, like we did for the Diagnostic icons